src/components/Blog/FeaturedArticle/FeaturedArticle.style.ts

```diff
@@ -2,8 +2,9 @@ import { urbanist } from '@/fonts/fonts';
 import type { BoxProps, Breakpoint, TypographyProps } from '@mui/material';
 import { Box, Skeleton, Typography, lighten } from '@mui/material';
 import { alpha, styled } from '@mui/material/styles';
+import Link from 'next/link';
 
-export const FeaturedArticleContainer = styled(Box, {
+export const FeaturedArticleContainer = styled(Link, {
```

Comment:
I think found this name to be a bit confusing - would FeaturedArticleLink be a better name now?